src/ducks/synths.ts

```diff
@@ -99,7 +97,7 @@ export const synthsSlice = createSlice({
 			const availableSynths: SynthDefinitionMap = keyBy(
 				synths.map((synth) => ({
 					...synth,
-					isFrozen: FROZEN_SYNTHS.includes(synth.name),
+					isFrozen: false,
```

Comment:
by default all synths are not frozen, and then updated as soon as we get them from the frozen list